config/vendor_api/v1.0.yml

```
@@ -503,6 +503,7 @@ components:
           type: array
           items:
             "$ref": "#/components/schemas/Reference"
+          description: References will only be populated once a candidate has accepted an offer and has received a reference.
```

Suggestion. “Populated” might get misinterpreted as being about the attributes of the references rather than whether they appear at all? Also maybe worth mentioning that there is now no limit to the number of references (previously it’d always contain exactly 2)?

```suggestion
          description: References will only be included once a candidate has accepted an offer and the reference has been received. There is no limit to the number of references.
```

---

good points frankie!